### PR TITLE
[Bundle][RU] actual translation

### PR DESCRIPTION
**Pull request by Remint32**

### CODE REVIEW

**Comment by XEN0PHIL** [comment]
А исправишь **block.segment.description**? :eyes: 
Лень создавать отдельный PR

File: core/assets/bundles/bundle_ru.properties [hunk 1]
```diff
@@ -1194,19 +1195,19 @@ block.unloader.description = Выгружает предметы из любог
...
-block.spectre.description = Массивная двуствольная пушка. Стреляет крупными бронебойными пулями по воздушным и наземным целям.
+block.spectre.description = Массивная двуствольная пушка. Стреляет крупными бронебойными снарядами по воздушным и наземным целям.
 block.meltdown.description = Массивная лазерная пушка. Заряжает и стреляет постоянным лазерным лучом в ближайших врагов. Требуется охлаждающая жидкость для работы.
 block.command-center.description = Командует перемещениями боевых единиц по всей карте.\nУказывает подразделениям [accent]собираться[] вокруг командного центра, [accent]атаковать[] вражеское ядро или [accent]отступать[] к ядру/фабрике. Если вражеское ядро отсутствует, единицы будут патрулировать при команде [accent]атаки[].
 block.repair-point.description = Непрерывно лечит ближайшую поврежденную боевую единицу или мех в своём радиусе.
-block.segment.description = Повреждает и разрушает приходящие снаряды. Не взаимодействует с лазерными лучами.
+block.segment.description = Повреждает и разрушает приближающиеся снаряды. Не взаимодействует с лазерными лучами.
```
Comment (Remint32) [inline core/assets/bundles/bundle_ru.properties:1213]: @XEN0PHIL 👌

**Comment by Remint32** [comment]
@Anuken Ready for merge

File: core/assets/bundles/bundle_ru.properties [hunk 2]
```diff
@@ -871,7 +871,7 @@ unit.eclipse.name = Затмение
 unit.mono.name = Моно
 unit.poly.name = Поли
 unit.mega.name = Мега
-unit.risse.name = Раскол
+unit.risse.name = Риссо
```
Comment (Anuken) [inline core/assets/bundles/bundle_ru.properties:874]: This should be *unit.risso.name*, I changed it.

Comment (Remint32) [reply core/assets/bundles/bundle_ru.properties]: https://github.com/Anuken/Mindustry/pull/2233/commits/da4f042fa4c4ecc8386c2dd991178cffc2c8c0fc

**Comment by Remint32** [comment]
Do Players Dream of Russian Translation?

**Pull request merged**